flow/layers/opacity_layer_unittests.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/flow/layers/opacity_layer.h"
```

Comment:
Github won't let me comment on an empty file, but I think you added an empty file `flow/layers/layer_tree_unittests.cc` to the PR.